lib/atp_scraper/activity.rb

```diff
@@ -43,15 +42,11 @@ def search_tournaments_doc(activity_doc)
       activity_doc.css(".activity-tournament-table")
     end
 
-    def search_records_doc(tournament_doc)
-      tournament_doc.css(".mega-table tbody tr")
-    end
-
-    def create_record(record, player, tournament)
+    def create_record(record, tournament)
       {
         year: tournament[:year],
         player_name: @player_name,
-        player_rank: player[:rank],
+        player_rank: tournament[:ranking],
```

Comment:
__[reek]__
* [FeatureEnvy] refers to tournament more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)